news.html

```diff
@@ -8,13 +8,21 @@
 
     <div class="container community-wrapper">
 
-        <div class="col-sm-8 wrapper">
+        <div class="col-sm-12 wrapper">
```

Comment:
@bsteinlo this causes the page to render with a huge amount of dead space to the right which makes the it look like something is missing from the page. Is that intentional?

<img width="1280" alt="screen shot 2016-10-25 at 21 59 53" src="https://cloud.githubusercontent.com/assets/49902/19702309/490b8ce2-9aff-11e6-82ad-d567f68d4977.png">